src/components/language-selector/language-selector.css

```diff
@@ -5,19 +5,27 @@
     height:  1.5rem;
 }
 
-.disabled {
-    opacity: .5;
-}
-
+/* Position the language select over the language icon, and make it transparent */
 .language-select {
-    margin: .5rem;
-    height: 1.85rem;
-    border: 1px solid $motion-primary;
+    position: absolute;
+    width: 3rem;
+    height: 3rem;
```

Comment:
For height, use $menu-bar-height from units.css
For width, this should always match language-menu in menu-bar.css. Maybe it would be good to add another constant for that in units.css to make sure they don't get out of sync.